### PR TITLE
Add --ember-path option to template generator.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

File: lib/generators/ember/template_generator.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ class TemplateGenerator < ::Rails::Generators::NamedBase
       source_root File.expand_path("../../templates", __FILE__)
 
       desc "Creates a new Ember.js template."
+      class_option :ember_path, :type => :string, :aliases => "-d", :default => false, :desc => "Custom ember app path"
```
Comment (bcardarella) [inline lib/generators/ember/template_generator.rb:11]: Does Rails use `-d` elsewhere for paths? I know `-d` is used by `rails new` for setting the database

Comment (rwjblue) [inline lib/generators/ember/template_generator.rb:11]: I am unsure.  This follows the current convention of `ember-rails` (and the rest of our generators).  We can certainly change it (or drop the abbreviated flag all together).

Comment (rwjblue) [inline lib/generators/ember/template_generator.rb:11]: @bcardarella - Completely up to you, I don't mind changing the others either...

Comment (bcardarella) [inline lib/generators/ember/template_generator.rb:11]: :+1: for following whatever ember-rails is doing

**Pull request merged**